### ISSUE TITLE
Add support for multiple authors

### ISSUE THREAD

**Issue by vchrombie** [Bitergia/release-tools#49]
This is a feature request to support updating multiple authors during notes generation.

This might be needed when more than one person works on an issue and they usually credit the other persons by adding the `Co-authored-by:` commit trailer. I think this could be extended to the changelog entries as well.

We can use a format something like below
```
title: 'Fix bug #666'
category: fixed
author: 
    - John Smith <jsmith@example.com>
    - John Doe <jdoe@example.com>
issue: 666
notes: >
  The bug was making it impossible to cast a spell on
  a magician.
```

There is another alternative, but I don't personally prefer but would like to listen to your thoughts as well.
```
author: John Smith <jsmith@example.com>, John Doe <jdoe@example.com>
```

@sduenas, what do you think about this feature?

**Comment by sduenas** [comment]
@vchrombie definitely, it will be great to have something like that. I agree I prefer first proposal.
